central/detection/lifecycle/manager_impl.go

```diff
@@ -248,6 +253,19 @@ func (m *managerImpl) buildMapAndCheckBaseline(indicatorSlice []*storage.Process
 	}
 }
 
+func (m *managerImpl) SendBaselineToSensor(pw *storage.ProcessBaseline) {
```

Comment:
biggest nit of all time (especially since I think this was just copied over).

`pw` doesn't really make much sense or tell us anything.  Recommend something like `baseline` or at least `pb`.

Reply:
I don't think this is the biggest nit of all time. `baseline` is a better variable name than `pw`. Even though it is copied it is good to make improvements.